tests/test_aaa_authorization_service.rb

```diff
@@ -22,54 +22,72 @@ class TestAaaAuthorizationService < CiscoTestCase
   def setup
     super
     feature_tacacs
+    preconfig_tacacs_server_access(tacacs_groups[0])
```

Comment:
I did not write the original method, but it seems that the difference is that `preconfig_tacacs_server_access` creates a `tacacs-server` that the `aaa group server` has access to.  Not sure why it's necessary, but I kept it in.